src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -439,10 +461,12 @@ public LoggingRule FindRuleByName(string ruleName)
         }
 
         /// <summary>
-        /// Removes the specified named logging rule.
+        /// Removes the logging rule with the specified name.
+        /// This method only works if the rule, which shall be removed, has a name.
```

Comment:
This is pretty much echoes the initial sentence, so doesn't feel needed. But I like the idea of mentioning `<see cref="LoggingRules.RuleName"/>`.

Reply:
Like I answered above, the name of the logging rule is essential for us. Therefore I would call it an exotic feature, but the only available way to identify and change existing logging rules. This should be reflected by the documentation. Maybe most users don't use it, and maybe they will never need it, but maybe they will see a use for it as soon as they know better about the purpose and the possibilities, and when they understand that NLog configuration is not that static as it appears to be from using a config file (which is read once in the beginning).